src/services/message.service.js

```diff
@@ -0,0 +1,22 @@
+const { EventEmitter } = require('events');
+const { Message } = require('../models/Message.model');
+
+const messageEmitter = new EventEmitter();
+
+const create = async (author, text, roomId) => {
+  const message = await Message.create({ author, text, roomId });
+
+  messageEmitter.emit('message', message);
+
+  return message;
+};
+
+const getByRoomId = async (id) => {
+  messageEmitter.once('message', async () => {
+    const roomMessages = await Message.findAll({ where: { roomId: id } });
+
+    return roomMessages;
+  });
+};
```

Comment:
The `getByRoomId` function is currently not returning any value. The `messageEmitter.once` listener will execute the callback when a 'message' event is emitted, but the function itself does not return the `roomMessages`. Consider restructuring the function to return the messages directly or use a different approach to handle the event.